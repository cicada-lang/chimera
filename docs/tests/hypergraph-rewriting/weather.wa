hyperrule weather {
  [SunRain()] => [Sun(), Rain()]
  [Rain()] => [WearUmbrella()]
  [Sun()] => [WearSunGlasses()]
}

eval weather(quote [SunRain()])
