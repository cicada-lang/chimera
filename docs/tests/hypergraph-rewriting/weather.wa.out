[WearUmbrella(), WearSunGlasses(), Cool()]
[InCar(), WearSunGlasses(), Cool()]
